app/Http/Controllers/Front/ClientController.php

```diff
@@ -703,6 +703,18 @@ public function profile()
         }
     }
 
+    public function generateMerchantRandomString($length = 10)
```

Comment:
we can rewrite this method to do the same thing in one line return substr(bin2hex(random_bytes($length)), 0, $length);